SubtitlesParser/Classes/Writers/SsaWriter.cs

```diff
@@ -73,8 +74,14 @@ private string SubtitleItemToDialogueLine(SubtitleItem subtitleItem)
             fields[5] = "0"; // left margin
             fields[6] = "0"; // right margin 
             fields[7] = "0"; // vertical margin
-            // combine all items in the `Lines` property into a single string, with each item being seperated by an SSA newline (\N) 
-            fields[9] = subtitleItem.Lines.Aggregate(string.Empty, (current, line) => current + $"{line}\\N").TrimEnd('\\', 'N'); // text
+
+            // combine all items in the `Lines` property into a single string, with each item being seperated by an SSA newline (\N)
+            // check if we should be including formatting or not (default to use formatting if plaintextlines isn't set) 
+            if (includeFormatting == false && subtitleItem.PlaintextLines != null)
+                fields[9] = subtitleItem.PlaintextLines.Aggregate(string.Empty, (current, line) => current + $"{line}\\N").TrimEnd('\\', 'N');
```

Comment:
The aggregation function is the same in both cases.
Can you avoid repeating some code with a logic along those lines:
`
var lines = includeFormatting == false && subtitleItem.PlaintextLines != null ? subtitleItem.PlaintextLines : subtitleItem.Lines;
fields[9] = lines.Aggregate(string.Empty, (current, line) => current + $"{line}\\N").TrimEnd('\\', 'N');
`

Reply:
Fixed in latest commit. Sorry for the delay.

Reply:
Thanks!